master/interact.html

```diff
@@ -358,9 +358,9 @@ <h2 id="PointerEvents">Pointer events</h2>
 
 <h2 id="pointer-processing">Hit-testing and processing order for user interface events</h2>
 
-<dl class='definitions'  data-dfn-type="dfn" data-export="">
+<dl class="definitions">
```

Comment:
IIRC, `data-dfn-type` specified on a `dl` marks child `dt`s to the same type. You could check the spec.json that bikeshed downloads and see if those items show up as `dfn`.

Reply:
wasn't the original issue that these DLs didn't have dfn? The one called out was the DL on https://svgwg.org/svg2-draft/render.html#Definitions which has this pattern. But maybe it was due to using data-exports rather than data-dfn-exports?

Reply:
Maybe. Don't know.